### PR TITLE
Level2

### PR DESCRIPTION
**Pull request by rocrisp**
Remove podstatus, add appstatus.
Upgrade application version thru CR

### CODE REVIEW

File: l5-operator/config/resources/bestie-deploy.yaml [hunk 1]
```diff
@@ -36,7 +33,6 @@ spec:
           value: app
         - name: FLASK_ENV
           value: development
-          #todo fixme
```
Comment (skattoju) [inline l5-operator/config/resources/bestie-deploy.yaml:39]: should leave this in we should not have secrets in git

File: l5-operator/controllers/commoncode.go [hunk 2]
```diff
@@ -86,3 +90,185 @@ func (r *BestieReconciler) applyManifests(ctx context.Context, req ctrl.Request,
...
+					log.Error(err, "Failed to update bestie application status")
+					return err
+				}
+			}
+		} else {
+			fmt.Println("The substring is not present in the string.")
+		}
+	}
+	return nil
+}
```
Comment (skattoju) [inline l5-operator/controllers/commoncode.go:274]: missing new line at end of file

File: l5-operator/config/resources/bestie-deploy.yaml (hunk 1 above)
Comment (rocrisp) [reply l5-operator/config/resources/bestie-deploy.yaml:39]: I added it back

File: l5-operator/controllers/commoncode.go (hunk 2 above)
Comment (rocrisp) [reply l5-operator/controllers/commoncode.go]: added a new line.

**Pull request merged**